clients/benchmarks/dgemm_genperf.py

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/python
```

Comment:
This generates a representative set of DGEMM tests combinatorially, generating a YAML file. Duplicates are removed. The YAML file is > 200 MB, so it's preferable that it not be checked in but rather generated on demand.